backend/homework/5/Logback/src/main/java/org/example/Logging.java

```diff
@@ -0,0 +1,13 @@
+package org.example;
+
+import java.util.logging.Logger;
+
+public class Logging {
+    private static final Logger log = Logger.getLogger(Logging.class.getName());
+    public static void print(String s){
+        log.info(s);
```

Comment:
If this is so tightly coupled to info level only why not get rid of this entire class and use only 
`private static final Logger log = Logger.getLogger(Logging.class.getName());` inside classes where we want logging as this class is not doing anything significant.

Reply:
Also didn't find exception handling anywhere. Try adding those